src/components/Navbar.tsx

```diff
@@ -1,129 +1,161 @@
 import Link from "next/link";
```

Comment:
Think we can use Prettier to format this document now.

Reply:
My problem with this is that prettier thinks it's reasonable to format code like this:
![image](https://user-images.githubusercontent.com/32957299/230083001-f2aa5ab2-ad34-472a-95d5-b67dbb1b6ca5.png)
Especially since the indentation looked reasonable before formatting